sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 ### Breaking Changes
 
+- Deprecated KeyEncryptionAlgorithm::CKM_RSA_AES_KEY_WRAP in favor of KeyEncryptionAlgorithm::CkmRsaAesKeyWrap;
+
+- Deprecated KeyEncryptionAlgorithm::RSA_AES_KEY_WRAP_256 in favor of KeyEncryptionAlgorithm::RsaAesKeyWrap256;
+
+- Deprecated KeyEncryptionAlgorithm::RSA_AES_KEY_WRAP_384 in favor of KeyEncryptionAlgorithm::RsaAesKeyWrap384;

```

Comment:
We typically don't have new lines between the list of changes (I am not sure if it would break any of the parsing scripts on release).

Also, suggesting to add back-ticks along with period instead of semi-colons.

```suggestion
- Deprecated `KeyEncryptionAlgorithm::CKM_RSA_AES_KEY_WRAP` in favor of `KeyEncryptionAlgorithm::CkmRsaAesKeyWrap`.
- Deprecated `KeyEncryptionAlgorithm::RSA_AES_KEY_WRAP_256` in favor of `KeyEncryptionAlgorithm::RsaAesKeyWrap256`.
- Deprecated `KeyEncryptionAlgorithm::RSA_AES_KEY_WRAP_384` in favor of `KeyEncryptionAlgorithm::RsaAesKeyWrap384`.
```

Reply:
thank you for the code snippet, merged it, looks better than my version :)